tools/crs_converter/CRSconverter.xml

```diff
@@ -0,0 +1,316 @@
+                <assert_contents>
+            	    <has_size value="47450" delta="0"/>
+            	</assert_contents>
+            </output>
+            <output file='results/TIFF_output.tiff' name='tiff_output'>
+                <assert_contents>
+            	    <has_size value="691360" delta="0"/>
+            	</assert_contents>
+            </output>
+            <output file='results/BMP_output.bmp' name='bmp_output'>
+                <assert_contents>
+            	    <has_size value="231478" delta="0"/>
+            	</assert_contents>
+            </output>
+            <output file='results/Shapefile_dataset_shp.html' name='shapefile' compare='sim_size' delta="0">
+                <extra_files type='file' name='shapefile.shp' value='results/shapefile.shp' compare='sim_size' delta="0"/>
+                <extra_files type='file' name='shapefile.shx' value='results/shapefile.shx' compare='sim_size' delta="0"/>
+                <extra_files type='file' name='shapefile.dbf' value='results/shapefile.dbf' compare='sim_size' delta="0"/>
+                <extra_files type='file' name='shapefile.prj' value='results/shapefile.prj' compare='sim_size' delta="0"/>
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+.. class:: infomark
+
+**Uploading composite data**
+
+To upload a shapefile on Galaxy : Click on "..." (Browse or update dataset) or "Upload File" on the side bar > Upload > Composite > composite type : shp > upload the available files of your dataset > start > close 
+
+.. class:: infomark
+
+**What it does ?**
+
+This tool convert coordinates reference system of shapefiles (shp) for geospatial analysis. 
+Multiples CRS are available for conversion, if you are looking to convert your sapefile in a non-available CRS, feel free to open an issue on the Galaxy Ecology GitHub : https://github.com/galaxyecology/tools-ecology/issues
+
+.. class:: infomark
+
+**Input**
+
+This tool only takes shapefiles(shp) as input, if you need to convert a csv file to a shp file to use this tool, make sure to preserve the coordinates reference system of your file.
+
+.. class:: infomark
+
+**Output**
+
+This tool provides multiples possible outputs, mainly shapefiles, pdf and tiff.
+
+.. class:: infomark
+
+**More on datums and ellipsoids**
+
+An ellipsoid is a mathematically defined surface which approximates the geoid: the surface of the Earth's gravity field, which is approximately the same as mean sea level.
+The Datum is the center of the geoid.
+Depending on Earth's relief on a local studied area, or on one's criteria for earth center, Datum and Ellipsoid may vary.
+PROJ4 present 10 built-in Ellipsoids and Datums:  GRS80, airy, bessel, clrk66, intl, WGS60, WGS66, WGS72, WGS84, sphere.
+
+https://proj.org/en/9.4/usage/ellipsoids.html#ellipsoids
+
+Other usefull links:
+
+https://proj.org/en/9.4/usage/projections.html
+https://proj.org/en/9.4/geodesic.html
+
+https://proj.org/en/9.4/operations/projections/index.html
```

Comment:
And same here put as I did before this way the user just have to click on the sentence and it open the page directly :)